Gems/Atom/Feature/Common/Code/Source/CoreLights/PolygonLightFeatureProcessor.cpp

```diff
@@ -289,7 +289,7 @@ namespace AZ::Render
         else
         {
             AZ::Vector3 normal = AZ::Vector3::CreateFromFloat3(data.m_direction.data());
-            bounds.emplace<Hemisphere>(AZ::Hemisphere(position, radius, normal));
+            bounds.emplace<Hemisphere>(AZ::Hemisphere(position, radius, normal.GetNormalized()));
```

Comment:
This really should be done in `SetPolygonPoints()`, not here. That way the normalize call only happen when it needs to.